src/docs/style/wiredtiger11.css

```diff
@@ -0,0 +1,187 @@
+/* WiredTiger-specific styling. */
+
+@media (prefers-color-scheme: dark) {
+  body {
+    --page-foreground-color: #CCC;
+    --page-link-color: #00F;
+    --page-visited-link-color: #88F;
+  }
```

Comment:
Please explicitly specify the background color in the `body` element.

The currently deployed version of the page (https://source.wiredtiger.com/develop/index.html) still uses white background in my browsers (both Firefox and Safari), making the page hard to read.

Reply:
This is done automatically by Doxygen 9.11.

The deployed version is built by an older version which doesn't support dark mode.